keyboards/gmmk/pro/ansi/ansi.c

```diff
@@ -0,0 +1,238 @@
+/* Copyright 2021 Gigahawk
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ansi.h"
+
+#ifdef RGB_MATRIX_ENABLE
+led_config_t g_led_config = {{
+                                 // clang-format off
+    { 4,      NO_LED, NO_LED, 95,     65,     79, 5,      28 },
+    { 8,      2,      9,      0,      10,     75, 1,      7  },
+    { 14,     3,      15,     NO_LED, 16,     86, 6,      13 },
+    { 20,     18,     21,     23,     22,     94, 12,     19 },
+    { 25,     30,     26,     31,     27,     32, 29,     24 },
+    { 41,     36,     42,     37,     43,     38, 35,     40 },
+    { 46,     89,     47,     34,     48,     72, 78,     45 },
+    { 52,     39,     53,     97,     54,     82, 44,     51 },
+    { 58,     63,     59,     64,     NO_LED, 60, 62,     57 },
+    { 11,     90,     55,     17,     33,     49, NO_LED, 69 },
+    { NO_LED, 85,     93,     61,     96,     66, 50,     56 }
+                                 // clang-format on
+                             },
+                             {
```

Comment:
@drashna  Where should the `clang-format` guards be for this?

Reply:
ideally, right after the if-def